hal/targets/hal/TARGET_STM/TARGET_STM32L1/rtc_api.c

```diff
@@ -37,12 +38,30 @@
 static int rtc_inited = 0;
 #endif
 
-RTC_HandleTypeDef RtcHandle;
+static RTC_HandleTypeDef RtcHandle;
+
+#if DEVICE_RTC_LSI
```

Comment:
If just a regular macro, can we reserve the DEVICE name to be magically applied by device_has which should be a controlled list for now?